pallets/did/src/did_details.rs

```diff
@@ -21,21 +21,23 @@ use frame_support::{
 	ensure,
 	storage::{bounded_btree_map::BoundedBTreeMap, bounded_btree_set::BoundedBTreeSet},
 	traits::Get,
+	RuntimeDebug,
 };
-use kilt_support::deposit::Deposit;
 use scale_info::TypeInfo;
 use sp_core::{ecdsa, ed25519, sr25519};
 use sp_runtime::{traits::Verify, MultiSignature, SaturatedConversion};
 use sp_std::{convert::TryInto, vec::Vec};
 
+use kilt_support::deposit::Deposit;
+
 use crate::{
 	errors::{DidError, InputError, SignatureError, StorageError},
 	service_endpoints::DidEndpoint,
 	utils, AccountIdOf, BalanceOf, BlockNumberOf, Config, DidCallableOf, DidIdentifierOf, KeyIdOf, Payload,
 };
 
 /// Types of verification keys a DID can control.
-#[derive(Clone, Decode, Debug, Encode, Eq, Ord, PartialEq, PartialOrd, TypeInfo, MaxEncodedLen)]
+#[derive(Clone, Decode, RuntimeDebug, Encode, Eq, Ord, PartialEq, PartialOrd, TypeInfo, MaxEncodedLen)]
```

Comment:
Ah nice. I suppose we can get rid of other custom debug impls then as well, e.g. in parachain staking.

Reply:
yes. Thought it might not work, but it does. :) [b6542ca](https://github.com/KILTprotocol/mashnet-node/pull/360/commits/b6542ca4b12b7ddb7ac8284bcaef76e73e00c841)